pyproject.toml

```diff
@@ -17,6 +17,7 @@ pytorch-lightning = "^1.0.8"
 scipy = "^1.4.1"
 torch = "^1.7.0"
 fitsio = "^1.1.3"
+Sphinx = "^3.4.3"
```

Comment:
It'd be better to make Sphinx a development dependency (i.e., move it down a few lines in this file)